tool/truffle/jruby_truffle_runner/lib/jruby_truffle_runner.rb

```diff
@@ -0,0 +1,270 @@
+require 'optparse'
+require 'pp'
+require 'yaml'
+require 'fileutils'
+
+# TODO stored after run combinations like -S irb, -I test
```

Comment:
We've been annotating TODOs with initials or GitHub username & date to better track TODO ownership.